src/server/gc_worker/compaction_filter.rs

```diff
@@ -229,12 +280,17 @@ impl Drop for WriteCompactionFilter {
         }
 
         self.switch_key_metrics();
-        debug!(
-            "WriteCompactionFilter has filtered all key/value pairs";
-            "bottommost_level" => self.bottommost_level,
-            "versions" => self.total_versions,
-            "deleted" => self.total_deleted,
-        );
+        VERSIONS_AND_DELETES.with(|x| {
```

Comment:
shall we export the metrics to grafana instead of in info log?

Reply:
metrics are updated in `switch_key_metrics`.